package-lock.json

```diff
@@ -0,0 +1,18929 @@
+{
```

Comment:
I think this file should not be here since we're using `yarn`.
Having 2 lockfiles will certainly create conflicts

Reply:
done